app/views/inventories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+
+
+<p style="color: green"><%= notice %></p>
+<div class="card-header container text-center">
+  <h1>Inventories</h1>
+  <%= link_to "New inventory", new_inventory_path, data: { turbo: false },class: 'btn btn-dark btn-sm' %>
+</div>
+<% @inventories.each do |inventory| %>
+  <br>
+  <div class="container card mb-3" style="max-width: 540px;">
+  <div class="row g-0">
+    <div class="col-md-4 p-1">
+      <%= link_to "/inventories/#{inventory.id}", class: 'btn btn-warning p-2 mt-1' do %>
+      <h1><%= inventory.name %></h1>
+      <% end %>
+      <%= button_to "REMOVE", inventory, method: :delete, class:"btn btn-sm" %>
```

Comment:
- [ ] Please make sure that only inventory list can be removed by the user who created the inventory, currently, it can be deleted by anyone.